lib/travis/build/script/shared/rvm.rb

```diff
@@ -58,6 +63,11 @@ def ruby_version
             force_187_p371 vers
           end
 
+          def import_gpg_key
+            sh.cmd "command gpg2 --version &>/dev/null && gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys #{RVM_GPG_KEYS.join(" ")}",
```

Comment:
Should it be `gpg2 --keyserver ...`?

Reply:
Oops. You're right. (Though `gpg` seems to work as well.) I'll fix this. :-)